Platforms/QemuSbsaPkg/PlatformBuild.py

```diff
@@ -499,6 +515,51 @@ def ExtractFile(self, VirtualFilePath, HostFilePath:os.PathLike):
         ret = RunCmd("FileExtract", f"{VirtualFilePath} {HostFilePath} {self.path_to_vhd}")
         return ret
 
+class LinuxVirtualDriveManager(object):

```

Comment:
Would this be useful to have at a common python level (say in QemuPkg) so that we don't maintain multiple copies? Probably a lot of the python could be shared, so perhaps a separate PR could deal with all of it.